docs/howto/delete-app.txt

```diff
@@ -0,0 +1,22 @@
+==================================
+How to delete a Django application
+==================================
+
+Django provides the ability to group sets of features into Python packages
+called :doc:`applications</ref/applications/>`. However, when requirements
+change, apps may become obsolete or unnecessary. The following guide will help
+you delete an application safely.
+
+1. Remove all references to the app (i.e imports, foreign keys).
+
+2. Remove all models from your ``models.py`` file.
```

Comment:
I think saying "your models.py" could be confusing since a project may have many models.py that are "yours" for a developer.

```suggestion
2. Remove all models from the corresponding ``models.py`` file.
```

Reply:
Applied!